src/wmnds/patterns/banner/_banner.njk

```diff
@@ -1,13 +1,13 @@
 {% from "wmnds/components/phase-indicator/_phase-indicator.njk" import wmndsPhaseIndicator %}
 {% macro wmndsBanner(params) %}
 <div class="wmnds-grid wmnds-banner-container">
-   <div class="wmnds-col-auto wmnds-float-left wmnds-m-r-sm">
+   <div class="wmnds-col-auto wmnds-float-left ">
       {{
          wmndsPhaseIndicator() | indent(6) | trim
       }}
    </div>
    <div class="wmnds-col-auto">
-      <p>This is a new service - your <span class="wmnds-banner-container__feedback">feedback</span> will help us to improve it.</p>
+      <p>This is a new service - your <a href="#" class="wmnds-banner-container__feedback">feedback</a> will help us to improve it.</p>
```

Comment:
As the p has been styled, could you give it a BEM class such as
```.wmnds-banner-container__text```

Thanks!

Reply:
Cool, make sense, added